### DISCUSSION TITLE
Updated: /content/drive/MyDrive/ComfyUI/custom_nodes/ComfyUI-Impact-Pack Update done. send error: Cannot write to closing transport send error: Cannot write to closing transport

### DISCUSSION THREAD

**Discussion by Amit30swgoh** [Comfy-Org/ComfyUI-Manager#626] [General]
# ComfyUI-Manager: EXECUTE => ['/usr/bin/python3', '-m', 'pip', 'install', 'segment-anything']
 
## ComfyUI-Manager: EXECUTE => ['/usr/bin/python3', '-m', 'pip', 'install', 'piexif']
 [ComfyUI-Manager] skip black listed pip installation: 'transformers'

## ComfyUI-Manager: EXECUTE => ['/usr/bin/python3', '-m', 'pip', 'install', 'scipy>=1.11.4']
  Install: install script

### ComfyUI-Impact-Pack: Updating subpack
 [Impact Pack] EXECUTE: ['/usr/bin/python3', 'install.py'] in '/content/drive/MyDrive/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/impact_subpack'
 req_path: /content/drive/MyDrive/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/impact_subpack/requirements.txt
 req_path: /content/drive/MyDrive/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/impact_subpack/requirements.txt
 req_path: /content/drive/MyDrive/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/requirements.txt
 ### ComfyUI-Impact-Pack: Check basic models
Updated: /content/drive/MyDrive/ComfyUI/custom_nodes/ComfyUI-Impact-Pack
Update done.
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport
send error: Cannot write to closing transport

**Comment by ltdrdata** [comment]
It seems to be an issue occurring because you're using the Colab environment, so ComfyUI and the browser communicate through an unstable and slow tunneling connection.

**Reply by Amit30swgoh** [reply]
ok friend ty any better alterative? i though colab pro is good woth cloudflares perhaps alternative for cloudflare?  i like in colab that i can moun my google drive

**Comment by ltdrdata** [comment]
edit this code in `server.py` @ 37

```
async def send_socket_catch_exception(function, message):
    try:
        await function(message)
    except (aiohttp.ClientError, aiohttp.ClientPayloadError, ConnectionResetError) as err:
        logging.warning("send error: {}".format(err))
```

to 

```
async def send_socket_catch_exception(function, message):
    try:
        await function(message)
    except (aiohttp.ClientError, aiohttp.ClientPayloadError, ConnectionResetError) as err:
        print(f'message: {message}')
        logging.warning("send error: {}".format(err))
```

Then we can back track which message is the problem.
